Documentation/general/packaged-shim.md

```diff
@@ -0,0 +1,55 @@
+Packaged shim
+===============================
+
+For DotnetTools, if signed shim is required. You need to add the following extra property in you project file. Then run `dotnet pack`. After the nupkg is generated, shim is generated in the package. You can signed all the content of the nupkg which includes the shims.
```

Comment:
To create a signed package for your Dotnet Tool, you will need to create a signed shim. To do this, you need to add the following extra property in you project file:

```
   <PackAsToolShimRuntimeIdentifiers>[list of RIDs}</PackAsToolShimRuntimeIdentifiers>
```

When this property is set, `dotnet pack` will generate a shim in the package (nupkg). 

When the shim and other content is signed, you can sign the nupkg.